examples/cocoapods/CocoapodsExample/CocoapodsExample.xcodeproj/project.pbxproj

```diff
@@ -382,8 +382,8 @@
 				LD_RUNPATH_SEARCH_PATHS = "@executable_path/Frameworks";
 				"LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]" = "@executable_path/../Frameworks";
 				MACOSX_DEPLOYMENT_TARGET = 13.5;
-				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = com.newrelic.CocoapodsExample;
+				MARKETING_VERSION = 4.7;
+				PRODUCT_BUNDLE_IDENTIFIER = com.newrelic.NRApp.bitcode;
```

Comment:
I thought we removed bitcode?

Reply:
Yeah, bitcode support is gone. we haven't updated the bundleID that is used for testing app store submission though. I'll look into creating a ticket to fix this bundle ID, thank you.